### ISSUE TITLE
Accessing the scraped text?

### ISSUE THREAD

**Issue by SvanteRichter** [tardisx/linkwallet#6]
Would it be possible to add a way to access the scraped text? I see at https://github.com/tardisx/linkwallet/blob/8215eaa82168e555042919e1cdfe994a92c6fb9c/content/content.go#L26 that it is just appending all text elements together, but would it be possible to add a readability parser to get an article view saved? Basically I want to protect my bookmarks from link rot.

Thanks for the project!

**Comment by heliostatic** [comment]
Agreed, this would be great. Possibly pulling in the shiori go-readability library? https://github.com/go-shiori/go-readability
